changelog/unreleased/update-reva-to-1c425035c8a2.md

```diff
@@ -0,0 +1,17 @@
+Enhancement: Update reva to v1.4.0
```

Comment:
```suggestion
Enhancement: Update reva to v1.4.1
```

And now it is 1.4.1 !

Reply:
See comment below by @kulmann 
This can be updated when we finally get to a "version" for what will pass here.